jsonapi.gemspec

```diff
@@ -0,0 +1,16 @@
+version = File.expand_path('../JSONAPI_VERSION', __FILE__).strip
```

Comment:
Yeah but the `version.rb` is just a programmatic API for accessing the version through runtime (which currently does not seem to add much value to this gem). It's always possible to add it later.